server/src/main/java/org/jboss/as/server/mgmt/NativeManagementResourceDefinition.java

```diff
@@ -56,7 +56,6 @@
     public static final SimpleAttributeDefinition SOCKET_BINDING = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.SOCKET_BINDING, ModelType.STRING, false)
             .setXmlName(Attribute.NATIVE.getLocalName())
             .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, false))
-            .setAlternatives(ModelDescriptionConstants.INTERFACE)
```

Comment:
I don't see any uses of those on the server side. They are there on the host side, where they are the standard settings.

An HC process can use socket binding configs now, so we can look into adding that to the host management-interface resources. We'd need to deprecate and support the old style though, probably until EAP 8.